src/components/Btn/index.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from "prop-types";
+
+const Btn = ({ text, handleClick, styleBtn }) => {
```

Comment:
1. Never shortcut component's name!
2. don't shortcut prop's name.
3. Better to name `styles` instead of `styleBtn`